com.jetbrains.PyCharm

```diff
@@ -0,0 +1,94 @@
+{
+    "app-id": "com.jetbrains.PyCharm",
+    "runtime": "org.freedesktop.Platform",
```

Comment:
It probably wouldn't be bad idea to use Sdk for the runtime; I've never used Pycharm for longer than 10 minutes but I can imagine it has to build libraries that link with external C libs somehow. @TingPing, your thoughts?

Reply:
Yes using the Sdk would be useful (all other IDEs on here do it) and you get `git` from it also.